Makefile

```diff
@@ -0,0 +1,31 @@
+export V?=0
+
+CUR_PATH := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+.PHONY: all
+all:
+	$(MAKE) -C hello_world CROSS_COMPILE="$(HOST_CROSS_COMPILE)"
+	$(MAKE) -C random CROSS_COMPILE="$(TA_CROSS_COMPILE)"
+
+.PHONY: clean
+clean:
+	$(MAKE) -C hello_world clean
+	$(MAKE) -C random clean
+
+.PHONY: rootfs-list
+rootfs-list: fl:=$(GEN_ROOTFS_FILELIST)
+rootfs-list:
+	echo "Generating list of examples for rootfs, filelist = $(fl)..."
+	@echo "# filelist-tee-common /start" 				> $(fl)
+	@echo "dir /bin/examples 755 0 0" 				>> $(fl)
```

Comment:
I think in this case it will be better if all example apps have common prefix/suffix (something like `example_*` or `*_example`) to easily distunguish them from other CA binaries, and store them in /usr/bin, like you suggested
@jforissier what do you think?